### DISCUSSION TITLE
Specify a nodeSelector when customizing kube-prometheus

### DISCUSSION THREAD

**Discussion by Vivaldi59** [prometheus-operator/kube-prometheus#2322] [Q&A]
Hi, I am currently using the kube-prometheus package to monitor my kubernetess cluster. 

I decided to customize my package to have something more specific to my project, in doing so I wondered if there is any way for me to specify the node I want for each applucation to be deployed on. I went through the documentation, the pull requests and the discussion but I didn't really found something of the sort. 

So basically is there any way to specify a node for a specific application inside my Jsonnet?

Best Regards.
